content/W2016/W16-event-problem-solving-session.md

```diff
@@ -0,0 +1,17 @@
+title: Problem Solving Session
+Date: 2016-02-01
+Category: Events
+Tags: problem solving, computing, math
+Slug: problem-solving-w16
+Author: Nivasini Ananthakrishnan
+Summary: Evening of math and coding
+
+Thanks for coming out to the first problem solving session for this semester! We hope you had fun socializing through math and computing problems. If you missed it you can find the problems [here](https://drive.google.com/folderview?id=0B68pQQep_ETbeVg1NUUtUDdRNnc&usp=sharing). Or come visit us during our office hours for copies of the problem sets.
```

Comment:
So even though its already been in the past, you should make the event post in the present tense, as all the rest of them are. I know that seems weird but it makes things more consistent